ci-operator/config/openshift/sippy/openshift-sippy-master.yaml

```diff
@@ -3,6 +3,10 @@ base_images:
     name: release
     namespace: openshift
     tag: golang-1.15
+  golangci-lint:
+    name: golangci-lint
+    namespace: ci
+    tag: v1.37.1
```

Comment:
We moved to a custom build root (https://github.com/openshift/sippy/blob/master/Dockerfile.buildroot) for the eventual React frontend, I think you could add golangci-lint to it -- that way we can eventually run eslint from the same make target.

Reply:
Thanks for letting me know. I've updated the config to add the `make lint` step with the assumption that it is in the buildroot image. I've opened a separate PR (https://github.com/openshift/sippy/pull/256) to add `golangci-lint` to the buildroot image, which should be merged before this one.